hammers/query.py

```diff
@@ -157,8 +163,8 @@ def owned_ip_single(db, project_id):
          , status
          , {projcol} AS project_id
     FROM   neutron.floatingips
-    WHERE  {projcol} = %s;
-    '''.format(projcol=project_col(db.version))
+    WHERE  {projcol} = {project_id};
+    '''.format(projcol=project_col(db.version), project_id=project_id)
```

Comment:
Since project_id is set via args in the next line, you don't need to include it in format.